api/app/Mail/PasswordReset.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class PasswordReset extends Mailable
+{
+    use Queueable, SerializesModels;
```

Comment:
Is the PasswordReset send via Queue's? Otherwise I'll need to set a Queue worker on the server